test/functional/core/fileio_spec.lua

```diff
@@ -212,3 +215,63 @@ describe('tmpdir', function()
     ok(len > 4 and len < 256, '4 < len < 256', tostring(len))
   end)
 end)
+
+describe('skip do you really want to write on force write', function()
```

Comment:
Missing `before_each`
```suggestion
describe('skip do you really want to write on force write', function()
  before_each(clear)
```